File: CS-Examples/02_SMTP Client/SendMessages/SendMailsFromDisk.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email;
using Spire.Email.Smtp;

namespace SendMailsFromDisk
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void btnRun_Click(object sender, EventArgs e)
        {
            MailMessage msg = MailMessage.Load(this.filePath.Text,MailMessageFormat.Eml);
            SmtpClient client = new SmtpClient();
            client.Host = this.HostInput.Text;
            client.Port = int.Parse(this.PortInput.Text);
            client.Username = this.Username.Text;
            client.Password = this.Password.Text;
            client.ConnectionProtocols = Spire.Email.IMap.ConnectionProtocols.Ssl;
            try
            {
                client.SendOne(msg);
                MessageBox.Show("Sent Successfully");
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }
                   
        }
        private void LoadBtn_Click(object sender, EventArgs e)
        {
            this.openFileDialog1.Filter = "Mail files|*.eml";
            if (this.openFileDialog1.ShowDialog() == DialogResult.OK)
            {
                this.filePath.Text = this.openFileDialog1.FileName;

            }
        }
    }
    
}

File: VB-Examples/02_SMTP Client/SendMessages/SendMailsFromDisk.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email
Imports Spire.Email.Smtp

Namespace SendMailsFromDisk
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			Dim msg As MailMessage = MailMessage.Load(Me.filePath.Text,MailMessageFormat.Eml)
			Dim client As New SmtpClient()
			client.Host = Me.HostInput.Text
			client.Port = Integer.Parse(Me.PortInput.Text)
			client.Username = Me.Username.Text
			client.Password = Me.Password.Text
			client.ConnectionProtocols = Spire.Email.IMap.ConnectionProtocols.Ssl
			client.SendOne(msg)
		End Sub
		Private Sub LoadBtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles LoadBtn.Click
            Me.openFileDialog1.Filter = "Mail files|*.eml"
            If Me.openFileDialog1.ShowDialog() = Windows.Forms.DialogResult.OK Then
                Me.filePath.Text = Me.openFileDialog1.FileName

            End If
		End Sub
	End Class

End Namespace